migtests/tests/oracle/basic-live-test/validateAfterChanges

```diff
@@ -7,15 +7,26 @@ import collections
 import oracle
 
 def main():
+
 	parser = argparse.ArgumentParser(description='Process an argument.')
 	parser.add_argument('--ff_fb_enabled', type=str, help='An argument for checking the if fall-forward/fall-back is enabled or not')
+	parser.add_argument('--ff_enabled', type=str, help='An argument for checking the if fall-forward is enabled or not')
+	parser.add_argument('--fb_enabled', type=str, help='An argument for checking the if fall-back is enabled or not')
 	args = parser.parse_args()
 	ff_fb_enabled = args.ff_fb_enabled
-    
-	if ff_fb_enabled == 'true':
+	global ff_enabled
```

Comment:
Where are these global flags declared?

Reply:
oh are you just declaring it for the first time by specifying global? Didn't know you could do that 🤔

Reply:
Yes @priyanshi-yb had written the first cut of the parser. So following the same pattern.